flank-scripts/src/main/kotlin/flank/scripts/zenhub/ZenHubAPI.kt

```diff
@@ -8,11 +8,14 @@ import com.github.kittinunf.result.getOrNull
 import com.github.kittinunf.result.onError
 import com.github.kittinunf.result.success
 import flank.scripts.utils.toJson
+import flank.scripts.zenhub.objects.ConvertToEpicRequest
+import flank.scripts.zenhub.objects.UpdateEpicRequest
 import kotlinx.serialization.Serializable
 
 const val FLANK_REPO_ID = 84221974
 internal const val ZENHUB_BASE_URL = "https://api.zenhub.com/p1/repositories/$FLANK_REPO_ID"
 
+// GET
```

Comment:
Why not change the function name instead?

Reply:
Some method names clearly explain their purpose. Adding HTTP prefix name would look strange (in the end it is not HTTP client library) example: `convertIssueToEpic` -> `postConvertIssueToEpic`

But I think you are right and those `// [HTTP method]` comments are redundant. I'll remove them in `GitHubApi.kt` as well